cli/cmd/install.go

```diff
@@ -754,10 +753,8 @@ func render(w io.Writer, values *charts.Values, configs *pb.All) error {
 		return err
 	}
 
-	return processYAML(&buf, w, ioutil.Discard, resourceTransformerInject{
-		injectProxy: true,
-		configs:     configs,
-	})
+	_, err = w.Write(buf.Bytes())
```

Comment:
TIOLI: `processYAML()` ended the YAML blob with:
```
out.Write([]byte("---\n")) 
```
I think that's why all the unit tests have the extra:
```
- ---
```

Reply:
Right, `processYAML()` uses that to separate resources, even when there's only one. I think it's better to leave it out.